Checks/Timing/KiaiConsistencyCheck.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+using MapsetParser.objects;
+
+using MapsetVerifierFramework.objects;
+using MapsetVerifierFramework.objects.attributes;
+using MapsetVerifierFramework.objects.metadata;
+
+using MVTaikoChecks.Utils;
+
+using static MVTaikoChecks.Aliases.Mode;
+using static MVTaikoChecks.Aliases.Level;
+
+namespace MVTaikoChecks.Checks.Timing
+{
+    [Check]
+    public class KiaiConsistencyCheck : GeneralCheck
+    {
+        private const string _MINOR = nameof(_MINOR);
+
+        public override CheckMetadata GetMetadata() =>
+            new BeatmapCheckMetadata()
+            {
+                Author = "SN707, Nostril",
```

Comment:
aka SN wrote it and i roasted it

looks good to me if its good with you hivie

Reply:
didnt mean to open up a review lol

Reply:
you still effectively rewrote like half the code rofl >.<